packages/@aws-cdk/custom-resource-handlers/test/custom-resources-framework/expected/singleton-function-eval-nodejs.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable prettier/prettier,max-len */
+import * as path from "path";
+import { Construct } from "constructs";
+import * as lambda from "../../../aws-lambda";
+
+export class EvalNodejsSingletonFunction extends lambda.SingletonFunction {
```

Comment:
Just for my understanding, is this used in `packages/@aws-cdk/custom-resource-handlers/test/custom-resources-framework/expected/singleton-function-eval-nodejs.ts`? If not, what's the purpose of this?

Reply:
Yeah. It's just a template to use as "expected" to compare against a result in a unit test.